src/tsickle.ts

```diff
@@ -2156,6 +2156,14 @@ function addClutzAliases(
       if (declarations && !areAnyDeclarationsFromSourceFile(declarations, sf)) {
         continue;
       }
+
+      // default is a keyword in typescript, so the name of the export being default
+      // means that it's a default export
+      if (symbol.name === 'default') {
+        dtsFileContent += '// skipped emitting clutz aliases for a default export';
```

Comment:
Maybe make this comment say why? (e.g. "not supported yet" or some such?)

Reply:
Done!

Reply:
Looks like you need to rebaseline the test too.  (may as well add in the missing trailing newline while you're at it)